proto/prehog/v1alpha/teleport.proto

```diff
@@ -268,8 +268,6 @@ enum DiscoverResource {
   DISCOVER_RESOURCE_DOC_DATABASE_RDS_PROXY = 34;
   DISCOVER_RESOURCE_DOC_DATABASE_HIGH_AVAILABILITY = 35;
   DISCOVER_RESOURCE_DOC_DATABASE_DYNAMIC_REGISTRATION = 36;
-
-  DISCOVER_RESOURCE_SAML_APPLICATION = 37;
```

Comment:
Looks like the cloud side protos haven't had this added and when I've copied this across from cloud, it's had this line removed, @rudream is this intentional ? do we need to add this value in the cloud repo ?